### PR TITLE
Use HTML instead of Juno.span

### PR DESCRIPTION
**Pull request by malmaud**

### CODE REVIEW

File: REQUIRE [hunk 1]
```diff
@@ -12,3 +12,4 @@ Compat 0.18
 MacroTools 0.3.6
 AutoHashEquals 0.1.0
 MNIST 0.0.2
+Hiccup 0.1.0
```
Comment (gustafsson) [inline REQUIRE]: Is this row still needed?

Comment (malmaud) [reply REQUIRE]: Ah good catch.

**Comment by oxinabox** [comment]
Looks right to me, But I having checked it in Juno.
Have you @malmaud  ? If not I can do so later

**Comment by malmaud** [comment]
Get a chance to test yet?

**Comment by oxinabox** [comment]
Not, yet. I only have Juno (or local Julia at all), on one computer, and some else in the lab has been using all its RAM, so I've not been on it

**Comment by oxinabox** [comment]
Tested it, a space was missing, I am not 100% sure this fixes it but it should do

**Pull request merged**